### PR TITLE
Hints new design buttons

### PR DESCRIPTION
**Pull request by drbalar**
image
image

### CODE REVIEW

File: app/public/static/js/components/hint-button.js [hunk 1]
```diff
@@ -24,25 +24,25 @@ AFRAME.registerComponent("hint-button", {
...
+    // this.hintLabel.setAttribute("color", "black");
+    // this.hintLabel.setAttribute("scale", { x: 0.07, y: 0.07, z: 0.07 });
+    // this.hintLabel.setAttribute("align", "center");
+    // this.hintLabel.setAttribute(
+    //   "font",
+    //   "https://raw.githubusercontent.com/jaydhulia/aframe-fonts/master/fonts/poppins/Poppins-Bold.json"
+    // );
+    // this.hintLabel.setAttribute("shader", "msdf");
+    // this.hintLabel.object3D.position.set(x, y - (data.height * 2) / 2.6, z);
+    // this.el.appendChild(this.hintLabel);
```
Comment (ahmedhamodi) [inline app/public/static/js/components/hint-button.js:39]: Let's remove commented out code, unless this is needed for future (in which case let's leave a comment indicating that!)

Comment (drbalar) [reply app/public/static/js/components/hint-button.js]: oh yes. I forgot to remove

**Review by ahmedhamodi — APPROVED** [review]
lgtm, tested locally :)

Code approval. Pending design approval.

**Review by aaronabraham311 — APPROVED** [review]
Product & design approval

**Pull request merged**